Marlin/example_configurations/Cartesio/Configuration.h

```diff
@@ -1428,6 +1428,11 @@
 // With this option servos are powered only during movement, then turned off to prevent jitter.
 //#define DEACTIVATE_SERVOS_AFTER_MOVE
 
+// Volumetric filament diameter support
+// Activate this to use the DEFAULT_NOMINAL_FILAMENT_DIA as the default filament diameter
+// M200 D0 to disable
+//#define ENABLE_VOLUMETRIC_FILAMENT_DIAMETER
```

Comment:
The name I would suggest, simply describing the effect of the setting is `VOLUMETRIC_DEFAULT_ON` or `VOLUMETRIC_ON_BY_DEFAULT`. I prefer the shorter name.

In general, this feature is only filling in for boards lacking an EEPROM, so I don't expect a wide usage. We might want to move the setting to `Configuration_adv.h` to avoid adding yet another rarely-used option to the main config file.